### DISCUSSION TITLE
some actions document error

### DISCUSSION THREAD

**Discussion by digger-yu** [community/community#54497] [Actions] [answered]
### Select Topic Area

Question

### Body

https://docs.github.com/zh/actions/learn-github-actions/contexts
Not displayed properly in Simplified Chinese
as shown in Figure
![图片](https://user-images.githubusercontent.com/55081697/236363279-6ade7080-95a5-4663-a67a-eb23b2556970.png)
![image](https://user-images.githubusercontent.com/55081697/236364164-9c5cb372-3dd9-4921-841d-e1363644bafd.png)
Only English is currently displayed properly on this page

**Comment by jsoref** [comment, marked as answer]
It looks like the Markdown is corrupt.

There's a 👍 / 👎 at the bottom. I'd suggest using 👎 and explaining it's broken.
